.github/workflows/zulip_emoji_merge_delegate.yaml

```diff
@@ -0,0 +1,83 @@
+name: Zulip Emoji Merge Delegate
+
+on:
+  schedule:
+    - cron: '0 * * * *'  # Runs every hour
+
+jobs:
+  zulip-emoji-merge-delegate:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install zulip
+
+    - name: Run Zulip Emoji Merge Delegate Script
+      env:
+        ZULIP_API_KEY: ${{ secrets.ZULIP_API_KEY }}
+        ZULIP_EMAIL: ${{ secrets.ZULIP_EMAIL }}
+        ZULIP_SITE: https://leanprover.zulipchat.com
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        python - <<EOF
```

Comment:
Could you factor this script out into a separate file?

(Also, I guess you may already be aware, but secrets.ZULIP_EMAIL doesn't currently exist, we're just using `github-mathlib4-bot@leanprover.zulipchat.com` directly in our other scripts.)

Reply:
Yep, I'll work on this asap